web/wp-content/themes/dctx-v2/template-parts/content-blocks/block-wds-fifty-text-only.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-pgja03.php, but found phpcs-scan-pgjA03.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).